pkgs/os-specific/linux/mdadm/default.nix

```diff
@@ -46,7 +46,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Programs for managing RAID arrays under Linux";
-    homepage = "http://neil.brown.name/blog/mdadm";
+    homepage = "https://git.kernel.org/pub/scm/utils/mdadm/mdadm.git";
```

Comment:
https://raid.wiki.kernel.org/index.php/Linux_Raid

Reply:
I generally expect a homepage to at least contain a link to the source code.

Reply:
> I generally expect a homepage to at least contain a link to the source code. This looks more like a general RAID wiki.

(We are covering your needs here: https://github.com/NixOS/nixpkgs/issues/293838)